### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by AlbertoRodriguezm**
Done

### CODE REVIEW

File: Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.rdf [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	<hasMeasurement rdf:resource="Measurement8401"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Measurement8401">
+	<hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#int">29</hasTemperature>
+	<atTime rdf:resource="http://www.w3.org/2001/XMLSchema#dateTime">2010-06-12T12:00:12</atTime>
+    </rdf:Description>
+    <rdf:Description rdf:about="#User10A">
+	<hasName rdf:resource="Pedro"/>
```
Comment (ocorcho) [inline Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.rdf]: This should be a string

File: Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.ttl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+Class01	includes Sensor029, Computer101.
```
Comment (ocorcho) [inline Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.ttl]: Missing the base prefix.

File: Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.ttl [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+Class01	includes Sensor029, Computer101.
+Computer101 hasOwner User10A.
+Sensor029 hasMeasurement Measurement8401.
+User10An hasName Pedro.
```
Comment (ocorcho) [inline Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.ttl]: Pedro should be a string

File: Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.rdf [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    </rdf:Description>
+	<rdf:Description rdf:about="#Computer101">
+	<hasOwner rdf:resource="User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	<hasMeasurement rdf:resource="Measurement8401"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Measurement8401">
+	<hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#int">29</hasTemperature>
+	<atTime rdf:resource="http://www.w3.org/2001/XMLSchema#dateTime">2010-06-12T12:00:12</atTime>
```
Comment (ocorcho) [inline Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.rdf]: it should be rdf:datatype. Same below

Comment (AlbertoRodriguezm) [reply Assignment2/AlbertoRodriguezm-AlbertoRodriguezMartin.rdf]: Done

**Pull request merged**